homeassistant/components/mqtt/fan.py

```diff
@@ -341,6 +363,18 @@ def _prepare_subscribe_topics(self) -> None:
         """(Re)Subscribe to topics."""
         topics: dict[str, Any] = {}
 
+        def add_subscribe_topic(topic: str, msg_callback: MessageCallbackType) -> bool:
+            """Add a topic to subscribe to."""
+            has_topic = self._topic[topic] is not None
+            if has_topic:
```

Comment:
```suggestion
            if has_topic := (self._topic[topic] is not None):
```

Reply:
That was a python syntax trick that I was not aware of, thanks for the suggestion!